server/Dockerfile

```
@@ -0,0 +1,18 @@
+FROM node:14-alpine
+
+#Create app directory
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
```

don't run things as root on the container

```suggestion
USER www-data
```

---

I should note that USER must be put on top of the 3rd line: `RUN mkdir -p /usr/src/app` or the folder will be created as root